.travis.yml

```diff
@@ -35,14 +36,7 @@ before_script:
   - |
     if [[ ! -z "$WP_VERSION" ]] ; then
       bash bin/install-wp-tests.sh wordpress_test root '' localhost $WP_VERSION
-      if [[ ${TRAVIS_PHP_VERSION:0:2} == "5." ]]; then
-        mkdir -p $HOME/phpunit-bin
-        wget -O $HOME/phpunit-bin/phpunit https://phar.phpunit.de/phpunit-4.8.phar
-        chmod +x $HOME/phpunit-bin/phpunit
-        export PATH=$PATH:$HOME/phpunit-bin/
-      else
-        composer global require "phpunit/phpunit=5.7.*"
-      fi
+      bash bin/install-php-phpunit.sh
```

Comment:
Looks like this needs to be sourced rather than executed.  Also, error handling?  If this fails (and really the pre-existing script above as well), then it should fail the build.